test/PoolManager.t.sol

```diff
@@ -1000,12 +999,12 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
     }
 
     function test_collectProtocolFees_nativeToken_accumulateFees_gas() public {
-        uint16 protocolFee = 1028; // 00000100 00000100
+        uint24 protocolFee = 10242500; // 100111000100 100111000100
```

Comment:
Could you make this number a constant in the test file? `MAX_FEE_BOTH_TOKENS`? // 2500 2500